src/libs/libRocket/Include/Rocket/Core/StringBase.h

```diff
@@ -185,8 +189,18 @@ class StringBase
 	inline const T& operator[](size_type index) const;
 	inline T& operator[](size_type index);
 
+	// Offer c_str() to make the interface somewhat more consistent with std::string.
+	// It eases code migration to std::string in the future should that be needed.
+	// The name also imparts better expectations about what it does better
+	// than CString() which can be confused with things like MFC's class of the same name.
+	inline const T* c_str() const;
+
 protected:	
+	inline T* _LocalBuffer();
```

Comment:
It would be better. They seem to only use it when the caller has the same function name.